testutils/src/main/java/com/microsoft/identity/internal/testutils/kusto/CaptureKustoTestResultRule.java

```diff
@@ -79,4 +81,8 @@ public void evaluate() throws Throwable {
             }
         };
     }
+
+    private String sanitizeErrorMessage(@NonNull final String msg) {
+        return msg.replaceAll("\n", " ").replaceAll("\r"," ");
```

Comment:
I think you can use a character class here, and avoid creating two string instances.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1157/commits/32193bb3490e1b1b4ff0d55ca72dd0ef357d6e68